components/core/src/utils/make_dictionaries_readable/make-dictionaries-readable.cpp

```diff
@@ -51,8 +53,11 @@ int main (int argc, const char* argv[]) {
 
     // Write readable dictionary
     auto readable_logtype_dict_path = boost::filesystem::path(command_line_args.get_output_dir()) / streaming_archive::cLogTypeDictFilename;
+    auto readable_logtype_dict_index_path = boost::filesystem::path(command_line_args.get_output_dir()) / streaming_archive::cLogTypeSegmentIndexFilename;
```

Comment:
How about `readable_logtype_segment_index_path`  for consistency with the naming scheme throughout the rest of the code?